frontend/src/container/FormAlertRules/QuerySection.styles.scss

```diff
@@ -0,0 +1,35 @@
+.stage-run-query {
```

Comment:
This will be treated as a global class and should be moved inside the `.alert-tabs` class.

Reply:
done